src/test/java/seedu/commands/FindCommandTest.java

```diff
@@ -98,6 +101,27 @@ public void execute_invalidDateFormat_expectIllegalInputExceptionThrown() {
                 + System.lineSeparator(), outContent.toString());
     }
 
+    @Test
+    public void execute_longSpecificValue_expectIllegalInputExceptionThrown() {
+        commandInputForTest("find spending date 2024/10/10 extraString", incomes, spendings);
+        assertEquals(TAB + LONG_FIND_SPECIFIC_VALUE + FIND_COMMAND_FORMAT
+                + System.lineSeparator(), outContent.toString());
+    }
+
+    @Test
+    public void execute_longFromValue_expectIllegalInputExceptionThrown() {
+        commandInputForTest("find spending date 2024/10/10 extraString to 2024-12-12", incomes, spendings);
+        assertEquals(TAB + LONG_FIND_FROM_VALUE + FIND_COMMAND_FORMAT
+                + System.lineSeparator(), outContent.toString());
+    }
+
+    @Test
+    public void execute_longToValue_expectIllegalInputExceptionThrown() {
```

Comment:
are we standardising to type of message thrown for the exception thrown?

Reply:
Sure i can do it, I will standardise for all the other test case also then